checkers_bot_tournament/piece.py

```diff
@@ -1,8 +1,8 @@
-from typing import Tuple
+from typing import Tuple, Literal
 
 
 class Piece:
-    def __init__(self, position: Tuple[int, int], colour: str, is_king: bool = False):
+    def __init__(self, position: Tuple[int, int], colour: Literal['WHITE', 'BLACK'], is_king: bool = False):
```

Comment:
Maybe we can make an enum for the colours as well (similar to what you did for result).

Reply:
See #8